src/sq/intro.md

```diff
@@ -1,4 +1,4 @@
-Redis Stack offers an enhanced Redis experience via the following search and query features:
+Redis Stack offers an enhanced Redis experience via the following Query Engine features:
```

Comment:
Shouldn't the name be "Redis Query Engine". Not sure if it's okay to shorten it. Some comment for the below as well.